lib/numbers.md

```diff
@@ -5,20 +5,27 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Numbers like whole numbers and decimals are defined as fixnum or float numbers.
+For cases involving imaginary or other cases in math, Ruby handles them as seperate
+units as part of the fixnums or floats.
 
 # What are some common operations and comparisons you would perform on numbers?
```

Comment:
Besides the greater than/less than operators, there are some whose purpose is to return a fixnum rather than a true or false statement (like the <=> operator)